.travis.yml

```diff
@@ -1,11 +1,7 @@
 language: php
 
 php:
-  - '7.0'
-  - '7.1'
-  - '7.2'
-
-sudo: false
+  - '7.4'
```

Comment:
Did you pick 7.4 specifically, or can we support at least all 7.x versions?

In a client we try to offer a broader compatibility when possible, as we can't expect users to always have the luxury to jump to the latest version.

Reply:
As I mentioned to @aeden (in the previous comment https://github.com/dnsimple/dnsimple-php/pull/1#issuecomment-652238128) The rationale was to have typed attributes (and the fact that PHP 7.3 will be no longer supported by December 2020).

I do agree with the wider audience though...

Let me change this then... 👍

Reply:
https://www.php.net/supported-versions.php 7.2 seems reasonable